pkg/provisioner/ironic/ironic.go

```diff
@@ -84,6 +85,16 @@ func init() {
 		fmt.Fprintf(os.Stderr, "Cannot start: No IRONIC_INSPECTOR_ENDPOINT variable set")
 		os.Exit(1)
 	}
+	ironicTrustedCAFile = os.Getenv("IRONIC_CACERT_FILE")
```

Comment:
`IRONIC_CERT_FILE`. You don't need CACERT, at least if I understand your motivation right.

Reply:
We need a CA to be able to support the cert-manager certificate rotation. However,the user could also just provide the certificate itself.